js/security-headers.js

```diff
@@ -28,7 +29,7 @@ export default {
     ],
     connectSrc: [
       `'self'`,
-      env.PULSE_API || `https://network-pulse-api-staging.herokuapp.com/`
+      url.parse(env.PULSE_API).host || `https://network-pulse-api-staging.herokuapp.com/`
```

Comment:
@Pomax should this be `https://network-pulse-api-staging.herokuapp.com/` or `https://network-pulse-api-staging.herokuapp.com/api/pulse/`?